test/test_distribution.py

```diff
@@ -234,23 +241,24 @@ def dirichlet(concentration, value):
 
 @pytest.mark.parametrize('batch_shape', [(), (5,), (2, 3)], ids=str)
 @pytest.mark.parametrize('event_shape', [(1,), (4,), (5,)], ids=str)
+@pytest.mark.xfail(raises=AttributeError, reason="DirichletMultinomial is not implemented yet in NumPyro")
```

Comment:
nit: unless you're planning to implement `DirichletMultinomial` in NumPyro, can you change this `xfail` to check if `get_backend() != 'torch'` instead of `raises=AttributeError`?

Reply:
Oh, good idea. Thanks, it is better.

Reply:
good idea, even `@pytest.mark.skipif(get_backend() != 'torch')`